Ladybird/BrowserWindow.cpp

```diff
@@ -504,3 +506,19 @@ void BrowserWindow::copy_selected_text()
         clipboard->setText(qstring_from_ak_deprecated_string(text));
     }
 }
+
+bool BrowserWindow::eventFilter(QObject* obj, QEvent* event)
```

Comment:
Doesn't this affect "drag"? Why not filtering the middle button click instead?

Reply:
The drag is working properly, what do you mean for filtering for middle button?

Reply:
OK, no problem here, my "Qt" is a little rusty :)